### PR TITLE
Fixes deserialization of the VC request

### PR DESCRIPTION
**Pull request by NSeydoux**
The VC was deserialized expecting an URL-encoded value, instead of a base-64 encoded one.

- [X] I've added a unit test to test for potential regressions of this bug.
- [x] The changelog has been updated, if applicable.
- [X] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by matthieubosquet — APPROVED** [review]
LGTM

**Review by ThisIsMissEm — APPROVED** [review]
LGTM, but is this an interface that is defined in the spec anywhere?

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ The following changes have been implemented but not released yet:
   change is transparent to users.
 - `approveAccessRequest` wasn't using the default session from `@inrupt/solid-client-authn-browser`
   to set the ACR access appropriately, resulting in `401 Unauthenticated` errors.
+- `getAccessRequestFromRedirectUrl` was deserializing the received VC expecting
+an URL-encoded value, instead of a base64-encoded value, which is how it is serialized
+when redirecting the user to the VC management app.
```
Comment (ThisIsMissEm) [inline CHANGELOG.md:25]: make sure you run lint & prettier on this file

**Comment by NSeydoux** [comment]
@ThisIsMissEm no, this is purely internal really, this redirection-based flow is not something relying on a defined spec, which is why the query params managements has been absorbed in the API functions: devs should not have to meddle with these parameters manually if we can avoid it.

**Pull request merged**